tests/scancode/test_cli.py

```diff
@@ -452,3 +464,41 @@ def test_scan_can_run_from_other_directory():
         print(stderr)
     assert rc == 0
     check_scan(test_env.get_test_loc(expected_file), result_file, strip_dates=True, regen=False)
+
+
+class TestShortenFilename(TestCase):
+
+    def test_filename_is_shortened(self):
+        test = cli.shorten_filename(filename='012345678901234567890123.c')
+        expected = '0123456789...4567890123.c'
+        assert expected == test
+
```

Comment:
Use only one blank line between class methods. Two lines is only between top level module members